x-pack/plugins/cases/public/components/custom_fields/text/get_eui_table_column.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { EuiTableComputedColumnType } from '@elastic/eui';
+import type { CaseUI } from '../../../containers/types';
+import { getEmptyTagValue } from '../../empty_value';
+
+export const getEuiTableColumn = ({
+  key,
+  label,
+}: {
+  key: string;
+  label: string;
+}): EuiTableComputedColumnType<CaseUI> => ({
+  name: label,
+  render: (theCase: CaseUI) => {
+    const index = theCase.customFields.findIndex(
+      (element) => element.key === key && element.value !== null
+    );
+
+    if (index !== -1) {
+      return (
+        <p data-test-subj={`text-custom-field-column-view-${key}`}>
```

Comment:
Should we truncate the text?

**With truncation**

<img width="2282" alt="Screenshot 2023-10-30 at 12 54 29 PM" src="https://github.com/elastic/kibana/assets/7871006/99022ac2-465b-4442-9d36-f6e2d2efa421">

**Without truncation**

<img width="2274" alt="Screenshot 2023-10-30 at 12 54 04 PM" src="https://github.com/elastic/kibana/assets/7871006/82e98fd3-c0c8-49cc-94d8-79fc429316ed">